environment-docs.yml

```diff
@@ -10,3 +10,4 @@ dependencies:
 - ipython
 - xclim==0.24
 - matplotlib
+- xesmf>=0.5.3
```

Comment:
environment-docs.yml is only to build docs on ReadTheDocs.  Do we need xesmf to build docs (it wasn't there before)?  In general we want to keep this env file to the minimum to not burst the memory limit on ReadTheDocs.

Reply:
I see you're right! I didn't realize we didn't install finch on RTD.